### PR TITLE
DOC: Add updates in readme

### PR DESCRIPTION
**Pull request by aresnow1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ potential of cutting-edge AI models.
 <i><a href="https://join.slack.com/t/xorbitsio/shared_invite/zt-1z3zsm9ep-87yI9YZ_B79HLB2ccTq4WA">👉 Join our Slack community!</a></i>
 </div>
 
+## Updates
```
Comment (UranusSeven) [inline README.md:29]:
```suggestion
## Recent Updates
```

File: README_zh_CN.md [hunk 2]
```diff
@@ -25,6 +25,12 @@ Xorbits Inference（Xinference）是一个性能强大且功能全面的分布
 <i><a href="https://join.slack.com/t/xorbitsio/shared_invite/zt-1z3zsm9ep-87yI9YZ_B79HLB2ccTq4WA">👉 立刻加入我们的 Slack 社区!</a></i>
 </div>
 
+## 更新
```
Comment (UranusSeven) [inline README_zh_CN.md:28]:
```suggestion
## 最近更新
```

File: README.md [hunk 3]
```diff
@@ -26,6 +26,12 @@ potential of cutting-edge AI models.
...
+- Support fp4 and int8 quantization for pytorch model: https://github.com/xorbitsai/inference/pull/238
+- Add embedding APIs: https://github.com/xorbitsai/inference/pull/194
```
Comment (UranusSeven) [inline README.md:31]:
```suggestion
- Add embedding API: https://github.com/xorbitsai/inference/pull/194
```

File: README.md [hunk 4]
```diff
@@ -26,6 +26,12 @@ potential of cutting-edge AI models.
 <i><a href="https://join.slack.com/t/xorbitsio/shared_invite/zt-1z3zsm9ep-87yI9YZ_B79HLB2ccTq4WA">👉 Join our Slack community!</a></i>
 </div>
 
+## Updates
+- Support fp4 and int8 quantization for pytorch model: https://github.com/xorbitsai/inference/pull/238
```
Comment (UranusSeven) [inline README.md:30]:
```suggestion
- Support fp4 and int8 quantization for pytorch models: https://github.com/xorbitsai/inference/pull/238
```

File: README_zh_CN.md [hunk 5]
```diff
@@ -25,6 +25,12 @@ Xorbits Inference（Xinference）是一个性能强大且功能全面的分布
...
+- 支持 fp4 和 int8 量化的 PyTorch 模型: https://github.com/xorbitsai/inference/pull/238
```
Comment (UranusSeven) [inline README_zh_CN.md:29]:
```suggestion
- PyTorch 模型支持 4-bit 和 8-bit 量化: https://github.com/xorbitsai/inference/pull/238
```

**Comment by aresnow1** [comment]
There will be some changes, close it and open a new one later.

**Pull request closed without merging**